homeassistant/components/html5/notify.py

```diff
@@ -540,6 +544,11 @@ def _push_message(self, payload, **kwargs):
                     _LOGGER.error("Error saving registration")
                 else:
                     _LOGGER.info("Configuration saved")
+            elif response.status_code != 201:
+                _LOGGER.error(
+                    "There was an issue sending the notification:\n%s",
+                    response.text,
```

Comment:
Include status, or else it's a pain to debug.

```suggestion
                    "There was an issue sending the notification %s: %s",
                    response.status,
                    response.text,
```